src/app/tests/suites/certification/Test_TC_ACFREMON_1_1.yaml

```diff
@@ -79,7 +79,7 @@ tests:
       response:
           constraints:
               type: bitmap32
-              hasMasksSet: [0x3]
+              hasMasksSet: [0x4]
```

Comment:
Ah, yes, this definitely did not use to match the test plan...

Reply:
Oops posted a reply earlier that didn't publish. But looks like you caught it :) 

Also added a new test as well.